pkg/deploy/util/util.go

```diff
@@ -439,25 +523,25 @@ func HasSynced(dc *deployapi.DeploymentConfig, generation int64) bool {
 // IsOwnedByConfig checks whether the provided replication controller is part of a
 // deployment configuration.
 // TODO: Switch to use owner references once we got those working.
-func IsOwnedByConfig(deployment *api.ReplicationController) bool {
-	_, ok := deployment.Annotations[deployapi.DeploymentConfigAnnotation]
+func IsOwnedByConfig(obj metav1.Object) bool {
+	_, ok := obj.GetAnnotations()[deployapi.DeploymentConfigAnnotation]
```

Comment:
@tnozicka @mfojtik we need to deprecate the old way of determining ownership in favor of owner references.